### ISSUE TITLE
Namespace

### ISSUE THREAD

**Issue by pietercolpaert** [constraintAutomaton/shape-index-specification#2]
Let’s register a namespace for this (at this moment it has example.org)

Would w3id.org/shapeindex make sense?

See https://w3id.org/

**Comment by constraintAutomaton** [comment]
Yes, I guess we can do it when we get enough review? I think the namespace make sense.

**Comment by pietercolpaert** [comment]
Doesn’t hurt to already claim the namespace
